### PR TITLE
fixing links to monitoring U. lesson, adding links to the slack chann…

### PR DESCRIPTION
**Pull request by guy9**
…el and community forum, fixing typos

### CODE REVIEW

**Comment by guy9** [comment]
@annastuchlik please review

File: docs/source/troubleshooting/index.rst [hunk 1]
```diff
@@ -12,4 +12,5 @@ Troubleshooting Guide for Scylla Monitoring Stack
 Choose a topic to begin:
 
 * :doc:`Troubleshooting <monitor-troubleshoot>`
-* `Troubleshooting Guide for Scylla Manager and Scylla Monitor Integration <https://docs.scylladb.com/troubleshooting/manager_monitoring_integration/>`_
\ No newline at end of file
+* `Troubleshooting Guide for Scylla Manager and Scylla Monitor Integration <https://docs.scylladb.com/troubleshooting/manager_monitoring_integration/>`_
+* You can ask questions and discuss the ScyllaDB Monitoring Stack in the `ScyllaDB community forum<https://forum.scylladb.com/>`_ and on the `Slack channel<http://slack.scylladb.com/>`_.
```
Comment (annastuchlik) [inline docs/source/troubleshooting/index.rst]:
```suggestion
* You can ask questions and discuss the ScyllaDB Monitoring Stack in the `ScyllaDB community forum <https://forum.scylladb.com/>`_  and on the `Slack channel <http://slack.scylladb.com/>`_.
```

**Review by annastuchlik — CHANGES_REQUESTED** [review]
There are build errors because the links don't render correctly. 
The changes I suggested (adding space characters) will resolve the issues.

**Comment by guy9** [comment]
Fixed, thanks

**Review by annastuchlik — APPROVED** [review]
LGTM

**Comment by annastuchlik** [comment]
@amnonh Please merge.

**Pull request merged**